src/_guides/openapi/jsonpath.md

```diff
@@ -0,0 +1,359 @@
+| `[<selectors>]`   | child segment (Section 2.5.1): selects zero or more children of a node |
+| `.name`           | shorthand for ['name' ] |
+| `.*`              | shorthand for [*] |
+| `..⁠[<selectors>]` | descendant segment (Section 2.5.2): selects zero or more descendants of a node |
+| `..name`          | shorthand for .. [' name' ] |
+| `..*`             | shorthand for ..[*] |
+| `'name'`          | name selector (Section 2.3.1): selects a named child of an object |
+| `*`               | wildcard selector (Section 2.3.2): selects all children of a node |
+| `3`               | index selector (Section 2.3.3): selects an indexed child of an array (from 0) |
+| `0:100:5`         | array slice selector (Section 2.3.4): start:end:step for arrays |
+| `?<logical-expr>` | filter selector (Section 2.3.5): selects particular children using a logical expression |
+| `length(@.foo)`   | function extension (Section 2.4): invokes a function in a filter expression |
+
+_Overview of JSONPath Syntax, from [RFC 9535](https://www.rfc-editor.org/rfc/rfc9535#tbl-overview)._
+
+### Examples
+
+If that isn't making too much sense, here are some examples to help you visualize.
+
+| JSONPath               | Intended Result                                        |
+|------------------------|--------------------------------------------------------|
+| `$.store.book[*].author` | the authors of all books in the store                 |
+| `$..author`              | all authors                                            |
+| `$.store.*`              | all things in the store, which are some books and a red bicycle |
+| `$.store..price`         | the prices of everything in the store                  |
+| `$..book[2]`             | the third book                                         |
+| `$..book[2].author`      | the third book's author                                |
+| `$..book[2].publisher`   | empty result: the third book does not have a "publisher" member |
+| `$..book[-1]`            | the last book in order                                |
+| `$..book[0,1]`           | the first two books                                    |
+| `$..book[:2]`            | the first two books                                    |
+| `$..book[?@.isbn]`       | all books with an ISBN number                          |
+| `$..book[?@.price<10]`   | all books cheaper than 10                              |
+| `$..*`                   | all member values and array elements contained in the input value |
+
+_Example JSONPath Expressions and Their Intended Results When Applied to the Example JSON Value, from [RFC 9535: 1.5. JSONPath Examples](https://www.rfc-editor.org/rfc/rfc9535#name-jsonpath-examples)._
+
+By combining these bits of example syntax together you can do amazing and powerful things with JSONPath, so let's look at how to do those amazing things in OpenAPI.
+
+
+### JSONPath & OpenAPI
```

Comment:
What do you think about renaming this section “Tools” (or anything more fancy but that shares the intent of the paragraph: which is to share some useful tools used for the rest of the article.

Reply:
The goal of this section is more to move the narrative on from "JSONPath in General", to "JSONPath for OpenAPI", using some tools just to visualize the concepts instead of hand waving through them as some other articles have done. it could be `Using JSONPath with OpenAPI` or similar.